src/targets/objc/native.js

```diff
@@ -12,48 +14,74 @@ module.exports = function (options) {
   // Dependencies
   code.push('#import <Foundation/Foundation.h>');
   code.push(null);
-  code.push('NSURLSession *session = [NSURLSession sharedSession];');
-  code.push(null);
+
+  // Build headers (including cookies)
+  var headers = this.source.headers.reduce(function (headers, header) {
+    headers[header.name] = header.value;
+    return headers;
+  }, {});
+
+  if (this.source.cookies.length) {
+    var cookies = this.source.cookies.map(function (cookie) {
+      return encodeURIComponent(cookie.name) + '=' + encodeURIComponent(cookie.value);
+    });
+
+    headers['Cookie'] = cookies.join('; ');
+  }
+
+  if (Object.keys(headers).length) {
+    code.push(objcHelpers.nsDictionaryBuilder('headers', headers, opts.indent));
+  }
+
+  var parameters;
+
+  // Form-encoded body
+  if (this.source.postData.params && this.source.postData.params.length) {
```

Comment:
this is wrong, you need to check for `mimeType` here since it will be the same object for multipart